extra-games/gzdoom/autobuild/defines

```diff
@@ -0,0 +1,12 @@
+PKGNAME=gzdoom
+PKGSEC=games
+PKGDES="A feature centric port for all Doom engine games, based on ZDoom"
```

Comment:
What does "feature centric" even mean?

Reply:
This description was copied from upstream's, I think they want to say 'multi-features'......